Documentation/network/concepts/ipam/index.rst

```diff
@@ -26,6 +26,10 @@ Multiple CIDRs per node      ❌                     ❌                       
 Dynamic CIDR/IP allocation   ❌                     ❌                       ✅          ✅          ✅              ✅                ❌
 ============================ ====================== ======================== =========== =========== =============== ================= ===============
 
+Changing the IPAM mode of an existing cluster is impossible.
+You must reinstall Cilium to change the IPAM mode.
+Before reinstalling Cilium, delete all CiliumNode CRD objects.
```

Comment:
I struggle a little bit with the step to delete all CiliumNode CRD objects. Exactly when should they delete the CilumNode CRD objects? Before uninstalling? After uninstalling?

I think that the intent here is to state what is necessary to switch modes, but at the same time it also feels like it's half way between documenting limitations and trying to provide a migration guide. 

Strictly speaking this is probably correct, but also if we want to provide a path for users to change the IPAM mode then we could remove the need to delete CiliumNodeCRD objects in the implementation, and probably we would come up with a more comprehensive migration guide rather than dropping three terse sentences on how it could be done.

Reply:
Yeah, I agree.